docs/DEPLOYMENT.md

```
@@ -0,0 +1,268 @@
+
+```bash
+pm2 restart banking          # PM2
+sudo systemctl restart banking  # systemd
+docker restart banking        # Docker
+```
+
+## 10. Troubleshooting
+
+**Port already in use**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)